### ISSUE TITLE
Line Align breaking ASCII art

### ISSUE THREAD

**Issue by Shrinks99** [coldtype/st2#16]
Hello again! :)

I have a neat piece of ASCII art that I've generated it looks like this:

```
            :                                   
            -.                                  
            --:                                 
     .:-::-.=-=.        . :. .   : %%%%%%*=.    
   .-===-==:-:::.     ..-.=.== -..=@@@@@@@@@=   
  :::--:-:---:-:     ..:=::. =:- = @@@@@@@@@@*  
 :--=:...  ..-:      :=-..     .   ::::::=#@@%= 
 =-=:       ::      =--:                   =@%%.
:---        :      .-.:                     *%%=
:-=.               ::       -               :%%*
=-=                ::-     :-                %%#
===                :=-     --                %%%
=-=                -==    -=-                ###
+==                .:-    ..-                ###
==+                :--    @@%                ##*
-=+.               ::    :@@*               :#*+
:+==               .     *%%=      =        +**-
 +++-                   =%%%.     ==       -***.
 :+++=:............:::-*%%%=     -=+.....-=***- 
  =*+++*********######%%%%*     =+=+==++++***+  
   -+**********##########-     -+=++++++++++:   
    .:=+++++**********+-.       -=++=====-:     
                                 -=+            
                                  -+            
                                   :            
```

Unfortunately, ST2 doesn't seem to like this and the Line Align options seem to mess up the spacing, ignoring any spaces before characters?

![Screenshot 2023-02-10 151106](https://user-images.githubusercontent.com/5672810/218188380-01717abe-d873-43fd-a723-c97ef63937fe.png)

Here's the same text file with Line Align set to centred.

![Screenshot 2023-02-10 151215](https://user-images.githubusercontent.com/5672810/218188560-e9e25538-0c12-43fc-b109-8b7f4363e19b.png)

I hope I've nailed down this issue correctly... Note that this is tested with [Recursive](https://www.recursive.design/) which must have its mono axis set to 1 if you want this to have any hope of looking proper :)

When set to a different font (tested with Berkely Mono) the same issue occurs.

**Comment by stenson** [comment]
Ah, thanks for filing that! That makes sense, looks like the `StSt` call being used in ST2 doesn't have `strip=False` specified as an argument. (Default is `True`. Just kind of noting the details here for future reference.)

If you want a quick fix and are willing to poke around the source, you can search your local installed version of ST2 for `StSt` and add the `strip=False` kwarg and it should work correctly once you restart Blender.

Will close this issue once I've released a version w/ this change included.

**Comment by stenson** [comment]
Should be fixed now in the latest release (0.10) up on coldtype.xyz/st2
